index.js

```diff
@@ -134,6 +228,8 @@ function scan(render) {
   }
 
   function contains(value, el) {
+    // TODO: this is never going to be passed a vDOM and therefore will never be passed a
+    // mithril module, so how can onumloaders be added?
```

Comment:
mithril itself keeps track of unloading stuff. So this might be simplified. Calling `api.onunload` might result in a route change to an empty module or so. This would result in calling `onunload` to the root-module and all components automatically.